tournaments/index.js

```diff
@@ -1042,7 +1042,7 @@ let commands = {
 			if (tournament.banlist.length < 1) {
 				return this.errorReply("The tournament's banlist is empty.");
 			}
-			this.sendReplyBox("This tournament includes:<br />" + tournament.getBanlist());
+			this.sendReplyBox('<div class="infobox infobox-limited">This tournament includes:<br />' + tournament.getBanlist() + '</div>');
```

Comment:
You have 2 infoboxes with this setup. ``sendReplyBox`` nests the HTML in a infobox div.